lasagne/layers/recurrent.py

```diff
@@ -564,9 +550,9 @@ def get_output_for(self, input, mask=None, *args, **kwargs):
         # Because the input is given for all time steps, we can precompute
         # the inputs to the gates before scanning.
         # input is dimshuffled to (n_time_steps, n_batch, n_features)
-        # W_in_to_gates is (n_features, 4*num_units). input_dot_W is then
+        # W_in_to_gates is (n_features, 4*num_units). input_dot_w is then
```

Comment:
There actually is a flake8 plugin for that: https://pypi.python.org/pypi/pep8-naming
I can't find this convention in PEP8, though: https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments
It only tells which convention to use for functions, methods and variables, but not which one to use for function and method arguments.